osmoutils/osmocli/parsers.go

```diff
@@ -312,3 +316,12 @@ func ParseSdkInt(arg string, fieldName string) (sdk.Int, error) {
 	}
 	return i, nil
 }
+
+func ParseDec(arg, fieldName string) (types.Dec, error) {
+	fmt.Println(arg)
```

Comment:
Just to make sure this is removed before merge but please keep if it helps you debug before the PR is ready
```suggestion
```